packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -147,12 +147,15 @@ describe('ActorQueryOperationFilterSparqlee', () => {
       const output: IActorQueryOperationOutputBindings = <any> await actor.run(op);
       await new Promise<void>(resolve => output.bindingsStream.on('end', resolve));
       expect(logWarnSpy).toHaveBeenCalledTimes(3);
-      for (const call of logWarnSpy.mock.calls) {
+      logWarnSpy.mock.calls.forEach((call, index) => {
         const dataCB = <() => { error: any; bindings: Bindings }> call[2];
         const { error, bindings } = dataCB();
         expect(isExpressionError(error)).toBeTruthy();
-        expect(isBindings(bindings)).toBeTruthy();
-      }
+        const dataFact = new DataFactory();
```

Comment:
There's already a factory you can use here: https://github.com/comunica/comunica/blob/3889390dd0b21928e4683723b310000462f1c256/packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts#L14

Reply:
sorry